Functions.Templates/Templates/DurableFunctionsOrchestration-CSharp-Isolated/.template.config/template.json

```diff
@@ -0,0 +1,73 @@
+{
+  "author": "Microsoft",
+  "classifications": [
+    "Azure Function",
+    "Durable Functions Orchestration"
+  ],
+  "name": "DurableFunctionsOrchestration",
+  "identity": "Azure.Function.CSharp.Isolated.DurableFunctionsOrchestration.1.x",
+  "groupIdentity": "Azure.Function.DurableFunctionsOrchestration",
+  "shortName": "durable",
+  "tags": {
+    "language": "C#",
+    "type": "item"
+  },
+  "sourceName": "DurableFunctionsOrchestrationCSharp",
+  "symbols": {
+    "namespace": {
+      "description": "namespace for the generated code",
+      "replaces": "Company.Function",
+      "type": "parameter"
+    },
+    "HostIdentifier": {
+      "type": "bind",
+      "binding": "HostIdentifier"
+    }
+  },
+  "primaryOutputs": [
+    {
+      "path": "DurableFunctionsOrchestrationCSharp.cs"
+    }
+  ],
+  "defaultName": "DurableFunctionsOrchestrationCSharp",
+  "postActions": [
+    {
+      "Description": "Adding reference to Microsoft.Azure.Functions.Worker.Extensions.DurableTask NuGet package.",
+      "ActionId": "B17581D1-C5C9-4489-8F0A-004BE667B814",
+      "ContinueOnError": "true",
+      "ManualInstructions": [],
+      "args": {
+        "referenceType": "package",
+        "reference": "Microsoft.Azure.Functions.Worker.Extensions.DurableTask", "version": "1.0.0",
```

Comment:
I see this package is not yet published. Do we have planned date for publishing this?

Reply:
Yes - either later today or tomorrow.

Reply:
They are published now https://www.nuget.org/packages/Microsoft.Azure.Functions.Worker.Extensions.DurableTask/1.0.0